101-data-share-account/azuredeploy.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "accounts_template_name": {
+            "type": "String"
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "principalId": {
+            "type": "String"
+        },
+        "tenantId": {
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.DataShare/accounts",
+            "apiVersion": "2019-11-01",
+            "name": "[parameters('accounts_template_name')]",
+            "location": "[parameters('location')]",
+            "identity": {
```

Comment:
how is this used?  also, the property set doesn't look correct, on a systemAssigned msi the tenant and principal are created by the system, not supplied by the user

Reply:
removed systemassigned msi. user will enter only account name and location to deploy the template.